openedx_events/event_bus/avro/tests/test_avro.py

```diff
@@ -4,7 +4,11 @@
 
 from opaque_keys.edx.keys import CourseKey
 
-from openedx_events.enterprise import signals  # pylint: disable=unused-import
+# Each new folder with signals must be manually imported in order for the signals to be cached
+#   and used in the unit tests. Using 'disable=reimported' with pylint will work,
+#   because we just use the cached signal list.
+from openedx_events.content_authoring import signals  # pylint: disable=unused-import
+from openedx_events.enterprise import signals  # pylint: disable=reimported
```

Comment:
Nit: I'd add this to this line as well, but we'll see how strongly you disagree. ;)
```suggestion
from openedx_events.enterprise import signals  # See note above; pylint: disable=reimported
```

Reply:
I disagree strongly enough to not want to do it but not strongly enough to spend more than this comment arguing about it. I think it's unnecessary since it's just one line away.

Reply:
No worries. Skip it.